javaparser-core/src/main/javacc/java.jj

```diff
@@ -673,6 +673,63 @@ TOKEN :
  * Program structuring syntax follows.
  */
 
+ /**
+ * The stub files are represented by the {@link StubUnit}
+ */
+StubUnit StubUnit():
+{
+	/* Represents the list of compilation units that contained in a stub file */
+	NodeList<CompilationUnit> compilationUnits = emptyList();
+	CompilationUnit cu;
+
+	/*
+	* Represents the particular compilation unit components in each iteration of
+	* reading compilation units from the stub file
+	*/
+	PackageDeclaration pakage = null;
+    NodeList<ImportDeclaration> imports = emptyList();
+    ImportDeclaration in = null;
+    NodeList<TypeDeclaration<?>> types = emptyList();
+    ModifierHolder modifier;
+    TypeDeclaration tn = null;
+    ModuleDeclaration module = null;
+}
+{
+	( LOOKAHEAD(2) ";" )*
+	(
+	    /*
+	    * Reading the particular compilation unit in the stub file
```

Comment:
What do you mean by "particular" compilation units?  Do you just mean the ones in the file?
Minor (but aids readability): end a sentence with punctuation rather than making the reader guess where sentences end and start.